tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/functions/CustomFunctionTest.java

```diff
@@ -43,4 +43,12 @@ public void testDuplicateParameters() {
         BAssertUtil.validateError(compile, 0, "redeclared symbol 'param'", 1, 35);
     }
 
+    @Test(description = "Test defining ballerina function with extra parameters with missing comma")
+    public void testExtraParameters() {
+        CompileResult compile = BCompileUtil.compile("test-src/functions/extra-parameters.bal");
+        Assert.assertEquals(compile.getErrorCount(), 2);
+        BAssertUtil.validateError(compile, 0, "invalid operation: type 'byte' does not support indexing", 20, 13);
+        BAssertUtil.validateError(compile, 1, "missing key expr in member access expr", 20, 18);
```

Comment:
Shall we change this to use an index and validate the error count?

```suggestion
        int index = 0;
        BAssertUtil.validateError(compile, index++, "invalid operation: type 'byte' does not support indexing", 20, 13);
        BAssertUtil.validateError(compile, index++, "missing key expr in member access expr", 20, 18);
        Assert.assertEquals(compile.getErrorCount(), index);
```

That way we can validate that we only get the expected errors.